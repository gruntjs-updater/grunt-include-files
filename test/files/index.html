<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>HTML Index</title>
		
		<!-- Include stylesheets here -->
		<!-- include: "type": "css", "files": "<%= vars.cssPath %>/**/*.css" -->
		
		<!-- Include stylesheet generated by the main LESS (or SCSS) file. -->
		<link rel="stylesheet" type="text/css" href="main.css">
	</head>
	<body>
		<!-- Include html sources -->
		<!-- include: "type": "toc", "files": "<%=vars.htmlPath %>/**/*.html" -->

		<!-- Include javascript sources, using 'rename' option -->
		<!-- include: "type": "js", "files": "<%=vars.jsPath %>/src/main/webapp/resources/**/*.js" -->
		
		<!-- Include javascript sources -->
		<!-- include: "type": "js", "files": "<%= vars.jsPath %>/**/*.js" -->
		
		<!-- Include javascript sources, keep indentation but prevent it being multiple lines -->
		
		<!-- include: "type": "js", "files": "<%= vars.jsPath %>/**/*.js" -->
		
		<!-- Include javascript sources, but keep the include comment marker -->
		<!-- include: "type": "js", "files": "<%= vars.jsPath %>/**/*.js" -->
		(The included scripts would normally be here.)
		<!-- /include -->
		
		<!-- Include javascript sources, but using a different ordering -->
		<!-- include: "type": "js", "files": "<%= vars.jsPath %>/**/*.js", "ordering": "top-down" -->
		
		<!-- Include javascript sources, using an array -->
		<!-- include: "type": "js", "files": "<%= vars.jsArray %>" -->
		
		<!-- Include javascript sources, using an array of strings that also contains variables -->
		<!-- include: "type": "js", "files": ["js/_first.js","js/lib/**/*.js","<%= vars.jsPath %>/main.js"] -->
	</body>
</html>