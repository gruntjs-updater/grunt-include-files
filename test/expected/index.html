<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>HTML Index</title>
		
		<!-- Include stylesheets here -->
		<link href="css/lib/dep1.css" rel="stylesheet" type="text/css">
		<link href="css/lib/dep2.css" rel="stylesheet" type="text/css">
		<link href="css/style1.css" rel="stylesheet" type="text/css">
		<link href="css/style2.css" rel="stylesheet" type="text/css">
		
		<!-- Include stylesheet generated by the main LESS (or SCSS) file. -->
		<link rel="stylesheet" type="text/css" href="main.css">
	</head>
	<body>
		<!-- Include html sources -->
		<a href="html/main1.html">main1.html</a>
		<a href="html/main2.html">main2.html</a>
		<a href="html/sub/sub1.html">sub1.html</a>
		<a href="html/sub/sub2.html">sub2.html</a>

		<!-- Include javascript sources, using 'rename' option -->
		<script src="resources/common.js"></script>
		
		<!-- Include javascript sources -->
		<script src="js/_first.js"></script>
		<script src="js/lib/dep1.js"></script>
		<script src="js/lib/dep2.js"></script>
		<script src="js/main.js"></script>
		<script src="resources/common.js"></script>
		
		<!-- Include javascript sources, keep indentation but prevent it being multiple lines -->
		
		<script src="js/_first.js"></script>
		<script src="js/lib/dep1.js"></script>
		<script src="js/lib/dep2.js"></script>
		<script src="js/main.js"></script>
		<script src="resources/common.js"></script>
		
		<!-- Include javascript sources, but keep the include comment marker -->
		<!-- include: "type": "js", "files": "<%= vars.jsPath %>/**/*.js" -->
		<script src="js/_first.js"></script>
		<script src="js/lib/dep1.js"></script>
		<script src="js/lib/dep2.js"></script>
		<script src="js/main.js"></script>
		<script src="resources/common.js"></script>
		<!-- /include -->
		
		<!-- Include javascript sources, but using a different ordering -->
		<script src="js/_first.js"></script>
		<script src="js/main.js"></script>
		<script src="js/lib/dep1.js"></script>
		<script src="js/lib/dep2.js"></script>
		<script src="resources/common.js"></script>
		
		<!-- Include javascript sources, using an array -->
		<script src="js/_first.js"></script>
		<script src="js/lib/dep1.js"></script>
		<script src="js/lib/dep2.js"></script>
		<script src="js/main.js"></script>
		
		<!-- Include javascript sources, using an array of strings that also contains variables -->
		<script src="js/_first.js"></script>
		<script src="js/lib/dep1.js"></script>
		<script src="js/lib/dep2.js"></script>
		<script src="js/main.js"></script>
	</body>
</html>